docs/pages/challenges/micro/frontend/react2/challenge-task.md

```diff
@@ -0,0 +1,84 @@
+# React Challenge 2 🤯
+
+Welcome to the React coding challenge 2! In this challenge, your task is to build a frontend for a project management application using React. 
+We have already built a robust backend for your application service. 
+This challenge will test your understanding and application of React components, props, states, hooks and more. 
+
+## Requirements
+
+The application should be able to handle various projects and respective tasks. 
+
+Backend data model:
+-	Project properties: project_id, email_address, name, description, created_at
+-	Task properties: task_id, project_id, name, description, created_at, status (enum - OPEN / IN_PROGRESS / DONE)
+
+Project_id of project, task_id, created_at and email_address are automatically set upon creation and are not mutable. 
+
+### The following functionalities should be provided: 
+1.	Create a new project with a name and a description
+2.	Edit name and description of a project
+3.	Delete a project
+4.	Display all projects
+5.	Create a new task for an existing project with a project_id, name, description and status. The status is set to OPEN by default. 
+6.	Edit name, description and status of a task
```

Comment:
These are two separate functionalities - edit the name & description (most likely in some kind of edit dialog) and change the status (drag & drop in a Kanban board).